request/request.go

```diff
@@ -31,10 +81,26 @@ func (r *Request) GetQuery() map[string]interface{} {
 	return query
 }
 
+func (r *Request) GetHeader(k string) string {
+	r.ch <- fmt.Sprintf(`kong.request.get_header:%s`, k)
+	return <-r.ch
+}
+
 func (r *Request) GetHeaders() map[string]interface{} {
 	r.ch <- `kong.request.get_headers`
 	reply := <-r.ch
 	headers := make(map[string]interface{})
 	json.Unmarshal([]byte(reply), &headers)
 	return headers
 }
+
+func (r *Request) GetRawBody() string {
+	r.ch <- `kong.request.get_raw_body`
+	return <-r.ch
+}
+
+func (r *Request) GetBody() string {
+	r.ch <- `kong.request.get_raw_body`
```

Comment:
`kong.request.get_body` is a complicated function that takes optional arguments and does a lot. In the Lua version it decodes the body itself, but for calling it from Go we would have to run it in Lua, serialize the response and send it to Go, and then deserialize again. I'm wondering if it would make sense to actually reimplement it in Go based on the lower-level `get_raw_body` function or just provide the lower-level ones. I suggest making it a TODO for now until we figure this out.